src/core/db/repository/base.py

```diff
@@ -55,47 +56,47 @@ async def update(self, _id: int, instance: DatabaseModel) -> DatabaseModel:
         return instance  # noqa: R504
 
     @auto_commit
-    async def update_all(self, instances: list[dict]) -> list[DatabaseModel]:
+    async def update_all(self, instances: list[dict]) -> Sequence[DatabaseModel]:
         """Обновляет несколько измененных объектов модели в базе."""
         await self._session.execute(update(self._model), instances)
         return instances
 
-    async def get_all(self) -> list[DatabaseModel]:
+    async def get_all(self) -> Sequence[DatabaseModel]:
         """Возвращает все объекты модели из базы данных."""
-        objects = await self._session.execute(select(self._model))
-        return objects.scalars().all()
+        objects = await self._session.scalars(select(self._model))
+        return objects.all()
 
     @auto_commit
-    async def create_all(self, objects: list[DatabaseModel]) -> None:
+    async def create_all(self, objects: Sequence[DatabaseModel]) -> None:
         """Создает несколько объектов модели в базе данных."""
         self._session.add_all(objects)
 
     async def count_all(self) -> int:
         """Возвращает количество объектов модели в базе данных."""
-        objects = await self._session.execute(select(func.count()).select_from(self._model))
-        return objects.scalar()
+        objects = await self._session.scalar(select(func.count()).select_from(self._model))
+        return objects

```

Comment:
```suggestion
        return await self._session.scalar(select(func.count()).select_from(self._model))
```
`objects` - только запутает, он во множественном числе, хотя мы тут возвращаем только число.